src/templates/default_streamlit/snowflake.yml

```diff
@@ -0,0 +1,12 @@
+definition_version: 1
+streamlit:
+  name: streamlit_app
+  stage:
+  query_warehouse:
+  main_file: "streamlit_app.py"
+  env_file: environment.yml
+  pages_dir: pages/
+#  database:
+#  schema:
+#  additional_source_files:
+#    -
```

Comment:
@sfc-gh-turbaszek @sfc-gh-pjob Do you have some idea how it should looks like?

Reply:
```yml
database: my_streamlit_db
schema: my_streamlit_schema
additional_source_files:
  - common/hello.py
```

The `hello` should be used in `streamlit_app` to show imports of additional code. For db and schema we should do `create if not exists`.

Reply:
Added